### PR TITLE
:heavy_check_mark: Bookstore: initialize

### PR DESCRIPTION
**Pull request by mahabubx7**
### :heavy_check_mark: Bookstore: initialize
- [x] No linters error.
- [x] Maintained `git-flow` and `GitHub workflows`.
- [x] Created `React` app using `create-react-app`.
- [x] Created the `src/components` directory.
- [x]  Added `react-router` (v6) & implemented in `App.js`.
- [x] Added 2 routes for `/` & `/categories`. 
- [x] Created required components in the `src/components` directory.
  - [x] Made book state, as well as a button, to delete the book.
  - [x] Made a list to render individual books.
  - [x] Made a form to create new books
  - [x] Made a navigation to link to the 2 routes I created

### CODE REVIEW

File: src/components/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addBook(book);
+    setBook({
+      name: '',
+      author: '',
+      progress: 0,
+    });
+  };
```
Comment (iambenkis) [inline src/components/Form.jsx:26]: Good job completing this project 👍 by initializing components in an incredible way 👏 . When you add a new book it should be pushed to the array all along 💯 , I noticed that when you add a book , it is well pushed but if the user navigated through components 👍 , the `book`  array return to the initial state. Kindly solve that 😄 



https://user-images.githubusercontent.com/80458199/236176003-2ba03af0-cc40-4a3e-9d03-b905e4f05f99.mp4

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @mahabubx7 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.jsx (hunk 1 above)
Comment (mahabubx7) [reply src/components/Form.jsx:26]: @iambenkis  According to the requirement, I'm not asked to save the states in local Storage.

**Review by fobadara — APPROVED** [review]
Hi @mahabubx7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Paid attention to project requirements

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.jsx (hunk 1 above)
Comment (fobadara) [reply src/components/Form.jsx:26]: Sorry for the mix up

**Pull request merged**